### PR TITLE
config prow for https://github.com/GoogleCloudPlatform/netd

### PR DESCRIPTION
**Pull request by cadmuxe**

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -599,6 +600,17 @@ plugins:
   - size
   - welcome
 
+  GoogleCloudPlatform/netd:
```
Comment (fejta) [inline prow/plugins.yaml]: No cat?!

**Comment by cadmuxe** [comment]
@krzyzacy I guess trusted_org is for the k8s member, and it won't need the user to be k8s members, right?

File: prow/plugins.yaml [hunk 2]
```diff
@@ -16,6 +16,7 @@ triggers:
 - repos:
   - google/cadvisor
   - GoogleCloudPlatform/k8s-multicluster-ingress
+  - GoogleCloudPlatform/netd
```
Comment (krzyzacy) [inline prow/plugins.yaml]: re: I guess trusted_org is for the k8s member, and it won't need the user to be k8s members, right?

If you don't want this restriction, just drop the line.

File: prow/plugins.yaml (hunk 1 above)
Comment (krzyzacy) [reply prow/plugins.yaml]: :100: please consider add our best selling prow plugins:  `- cat` and `- dog`!

Comment (cadmuxe) [reply prow/plugins.yaml]: The cat plugin adds a cat image to an issue in response to the `/meow` command.
Aha, I love it.

File: prow/plugins.yaml (hunk 2 above)
Comment (cadmuxe) [reply prow/plugins.yaml]: oh, I noticed trusted_org for the previous repos section, didn't see this one also have it.
Thanks

**Pull request merged**